diff -Naur mtd/util/mkfs.jffs2.c mtd.modified/util/mkfs.jffs2.c
--- mtd/util/mkfs.jffs2.c	2020-05-22 09:14:15.201751811 -0700
+++ mtd.modified/util/mkfs.jffs2.c	2020-05-22 09:09:12.917757265 -0700
@@ -438,7 +438,11 @@
  */
 
 #ifdef __GNUC__
+#if __STDC_VERSION__ >= 199901L
+#define SCANF_PREFIX "m"
+#else
 #define SCANF_PREFIX "a"
+#endif
 #define SCANF_STRING(s) (&s)
 #define GETCWD_SIZE 0
 #else
